src/main/java/no/finn/unleash/util/UnleashConfig.java

```diff
@@ -88,8 +101,17 @@ public UnleashContextProvider getContextProvider() {
         return contextProvider;
     }
 
+    public static void setRequestProperties(HttpURLConnection connection, UnleashConfig config) {
+        connection.setRequestProperty(UNLEASH_APP_NAME_HEADER, config.getAppName());
+        connection.setRequestProperty(UNLEASH_INSTANCE_ID_HEADER, config.getInstanceId());
```

Comment:
Can this set `User-Agent` as well? Since this is getting a small refactor either way 🙂

Ref Unleash/unleash-client-node#39